.pre-commit-config.yaml

```
@@ -1,218 +1,43 @@
-  #     - id: samo-auto-fix-code-quality
-  #       name: SAMO-DL Auto-Fix Code Quality
-  #       entry: python scripts/maintenance/auto_fix_code_quality.py
-  #       language: python
-  #       types: [python]
-  #       pass_filenames: false
-  #       always_run: true
-
-# Global configuration
-default_language_version:
-  python: python3.8
+        args: [--ini, setup.cfg]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)